src/app/pages/ledger/connect-ledger-error/connect-ledger-error.layout.tsx

```diff
@@ -0,0 +1,75 @@
+import { FiCircle } from 'react-icons/fi';
+import { Box, color, Flex, Stack } from '@stacks/ui';
+
+import { BaseDrawer } from '@app/components/drawer';
+import { Caption } from '@app/components/typography';
+import ConnectLedgerError from '@assets/images/onboarding/ledger/connect-ledger-error.png';
+import { ErrorLabel } from '@app/components/error-label';
+import { PrimaryButton } from '@app/components/primary-button';
+import { Link } from '@app/components/link';
+
+import { ConnectLedgerTitle } from '../components/connect-ledger-title';
+
+interface PossibleReasonUnableToConnectProps {
+  text: string;
+}
+function PossibleReasonUnableToConnect(props: PossibleReasonUnableToConnectProps) {
+  const { text } = props;
+
+  return (
+    <Flex>
+      <Box mr="tight" mt="3px">
+        <FiCircle fill={color('text-caption')} size="4px" />
+      </Box>
+      <Caption>{text}</Caption>
+    </Flex>
+  );
+}
+
+interface ConnectLedgerErrorLayoutProps {
+  onCancelConnectLedger(): void;
+  onTryAgain(): void;
+}
+export function ConnectLedgerErrorLayout(props: ConnectLedgerErrorLayoutProps) {
+  const { onCancelConnectLedger, onTryAgain } = props;
+
+  return (
+    <BaseDrawer title={<Box />} isShowing onClose={onCancelConnectLedger}>
+      <Stack alignItems="center" pb="loose" px="loose" spacing="loose" textAlign="center">
+        <Box mt="tight" width="247px">
+          <img src={ConnectLedgerError} />
+        </Box>
+        <ConnectLedgerTitle />
+        <ErrorLabel fontSize="14px" lineHeight={1.4}>
+          Unable to connect
+        </ErrorLabel>
+        <Stack
+          border="2px solid"
```

Comment:
Same for this and the other hardcoded values. Does the system have variables for border width? I feel that I need a deep dive into the system's variable file.

Reply:
I don't think so, this is used everywhere in the wallet to create borders. If you search the code for `border=` this is how we do it across the app.

Reply:
got it! thanks